layout/common/article.jsx

```diff
@@ -47,7 +48,7 @@ module.exports = class extends Component {
                                 __html: _p('article.created_at', `<time dateTime="${date_xml(page.date)}" title="${new Date(page.date).toLocaleString()}">${date(page.date)}</time>`)
                             }}></span>}
                             {/* Last Update Date */}
-                            {page.updated && <span class="level-item" dangerouslySetInnerHTML={{
+                            {page.updated && !(skip_same_date && moment(page.date).isSame(moment(page.updated))) && <span class="level-item" dangerouslySetInnerHTML={{
```

Comment:
Better to set up a new variable for `!(skip_same_date && moment(page.date).isSame(moment(page.updated)))` so this statement can be cleaner.

```javascript
const updateTime = article ? article.update_time : true;
const isUpdated = page.updated && !moment(page.date).isSame(moment(page.updated));
const showUpdateTime = page.updated && ((updateTime === 'auto' && isUpdated) || updateTime === true);
```